gcp-ts-docker-gcr-cloudrun/cloud-run-deploy/index.ts

```diff
@@ -0,0 +1,59 @@
+import * as pulumi from "@pulumi/pulumi";
+import * as gcp from "@pulumi/gcp";
+import * as docker from "@pulumi/docker";
+
+const location = gcp.config.region || "us-central1";
+
+const config = new pulumi.Config()
```

Comment:
You are missing a bunch of semicolons in code. We use linter for examples, so please run the linter until you have zero warnings there.

Reply:
Should be good now?